### ISSUE TITLE
Installation help

### ISSUE THREAD

**Issue by liangheng75** [learningmatter-mit/VOID#4]
Hello, firstly, I congratulate that your work published on Science. However, in repeating your work, I have two problems bothering me. The first problem is that the VOID software you used in your work needs to used with Zeo++. However, I have already updated GCC and relevant third-party libraries as required, but there are still installation problems. Can you give me some installation suggestions? Another question is what method you used for PCA. There is a big difference between the two sets of axis obtained by using the PCA method in sklearn and your work.

**Comment by dskoda** [comment]
Hello,

Yes, the Voronoi docking algorithm requires Zeo++ to be run. The version of GCC I tested the compilation against is listed [here](https://github.com/learningmatter-mit/VOID#zeo-dependency). For more information, please also check [this past issue](https://github.com/learningmatter-mit/VOID/issues/2). We do not offer support for compiling third-party libraries. Please contact the Zeo++ developers if you have questions regarding the installation beyond the ones shown here.

Also, issue #5 recently shows that the SVN links may not be available for the time being.

As for the PCA, the axes are dependent on the molecular conformer. Therefore, if you are finding differences between the values, it is likely that the molecular conformers are different - which is completely reasonable. In our work, we calculated them consistently, making it possible to compare axes across several molecules. The code I used to obtain the axes is the following:

```python
def get_axes(xyz):
    from sklearn.decomposition import IncrementalPCA

    pca = IncrementalPCA(2)
    uv = pca.fit(xyz).transform(xyz)

    length = uv.max(0) - uv.min(0)
    length.sort()

    return length[0], length[1]
```
